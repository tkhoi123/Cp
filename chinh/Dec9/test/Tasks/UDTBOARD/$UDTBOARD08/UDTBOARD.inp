1 23 52
1 5 1 5
1 15 1 23
1 11 1 21
1 12 1 13
1 5 1 9
1 6 1 6
1 8 1 18
1 1 1 12
1 16 1 16
1 21 1 23
1 9 1 9
1 13 1 15
1 17 1 22
1 13 1 16
1 3 1 15
1 11 1 18
1 9 1 9
1 3 1 15
1 20 1 22
1 1 1 1
1 6 1 6
1 5 1 16
1 11 1 20
1 2 1 13
1 3 1 6
1 7 1 17
1 6 1 8
1 1 1 14
1 18 1 23
1 9 1 18
1 12 1 20
1 11 1 12
1 1 1 23
1 15 1 18
1 20 1 20
1 5 1 23
1 5 1 6
1 7 1 7
1 7 1 19
1 8 1 8
1 13 1 22
1 11 1 13
1 9 1 21
1 2 1 8
1 9 1 17
1 5 1 8
1 21 1 23
1 17 1 20
1 20 1 21
1 9 1 9
1 16 1 17
1 9 1 19
