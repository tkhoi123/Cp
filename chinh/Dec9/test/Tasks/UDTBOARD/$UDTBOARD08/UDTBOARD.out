4 5 8 8 14 16 15 16 19 15 20 22 23 20 21 20 19 17 13 14 12 9 6 
